nimbus-core/src/main/java/com/antheminc/oss/nimbus/channel/web/WebActionController.java

```diff
@@ -190,10 +211,40 @@ else if(o.getValue() instanceof Param) {
 		return ResponseEntity.status(HttpStatus.OK).body(null);
 	}
 	
+	@RequestMapping(value=URI_PATTERN_P+"/event/upload",consumes = MediaType.MULTIPART_FORM_DATA_VALUE,  produces="application/json", method=RequestMethod.POST)
+	public Object handleUpload(HttpServletRequest req, @RequestParam("pfu") MultipartFile file, @RequestParam("domain") String domain) {
+		if (file.isEmpty()) {
+			return new Holder<>(false);
+		}
+		
+		Command command = this.builder.build(req);
+		Command uploadCommand = CommandBuilder.withPlatformRelativePath(command, Type.PlatformMarker, "/" + domain).getCommand();
+		uploadCommand.setRequestParams(command.getRequestParams());
+		uploadCommand.setAction(Action._new);
```

Comment:
This is hardcoding of only an new file upload scenario. How will the need to delete, replace and read file be handled?

Reply:
We don't intentionally keep the imported file. Ideally, our code should receive the file, if necessary save it temporarily, do an work and then remove the file if it was saved.  So the need to delete, replace, or read is not needed or supported.